packages.tsv

```diff
@@ -266,3 +266,4 @@ conda	piranha	1.2.1--gsl1.16_2	Piranha
 conda	trimmomatic	0.36--3	trimmomatic --help
 conda	bwa	0.7.15--0	which bwa
 conda	ribotaper	1.3.1--0	Ribotaper.sh 2>&1  | grep Usage > /dev/null
+conda	viral-ngs	1.10.1--2	illumina.py --help
```

Comment:
@tomkinsc the build number needs to encode also the proper build number: `1.10.1--py27_2`